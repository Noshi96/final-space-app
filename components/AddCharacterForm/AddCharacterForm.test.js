/* globals describe, it, expect  */

import { render } from '@testing-library/react'
import AddCharacterForm from './AddCharacterForm'

describe('AddCharacterForm', () => {
  it('renders form', () => {
    const cardDetailsModal = render(
      <AddCharacterForm isOpen closeModal={() => {}} />
    )
    expect(cardDetailsModal.container).toMatchSnapshot()
  })
})
