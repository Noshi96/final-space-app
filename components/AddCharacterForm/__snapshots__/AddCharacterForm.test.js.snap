// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddCharacterForm renders form 1`] = `<div />`;
